### PR TITLE
Issue #615 Store operations statistics

### PR DESCRIPTION
**Pull request by ljacomet**
This PR creates statistics for the store operations. It does not yet resolve the creation of a compound counter to expose a usefulness statistic from a given tier.

### CODE REVIEW

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
test this please

File: impl/src/main/java/org/ehcache/internal/store/offheap/AbstractOffHeapStore.java [hunk 1]
```diff
@@ -79,42 +96,50 @@ public AbstractOffHeapStore(String statisticsTag, Configuration<K, V> config, Ti
...
         @Override
         public OffHeapValueHolder<V> apply(K mappedKey, OffHeapValueHolder<V> mappedValue) {
           long now = timeSource.getTimeMillis();
 
-          if (mappedValue == null || mappedValue.isExpired(now, TimeUnit.MILLISECONDS)) {
-            if (mappedValue != null) {
-              onExpiration(mappedKey, mappedValue);
-            }
-            return null;
+        if (mappedValue == null || mappedValue.isExpired(now, TimeUnit.MILLISECONDS)) {
```
Comment (chrisdennis) [inline impl/src/main/java/org/ehcache/internal/store/offheap/AbstractOffHeapStore.java]: The indentation here is wrong.

**Pull request merged**